### ISSUE TITLE
[Feature Request] Properties on exported code

### ISSUE THREAD

**Issue by volkandkaya** [bernaferrari/FigmaToCode#96]
data-options='{"Size": "lg", "Color": "primary"}'

This would be amazing. I'm already heavily using the "Layer names" feature.

**Comment by bernaferrari** [comment]
How do you think I could know what's primary? Via the new variables thing?

**Comment by volkandkaya** [comment]
Yes, I would take a look at https://www.untitledui.com/free-figma-ui-kit it has a ton of components to test from.

![image](https://github.com/bernaferrari/FigmaToCode/assets/12369050/57071f04-1804-45e1-aabe-8b011e649e56)
